Jenkinsfile

```diff
@@ -1 +1,6 @@
-buildPlugin()
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+def configurations = [
```

Comment:
this change won't actually be run on this PR as I don't have write access to the repo, but I've ran the build locally and it passes now

Reply:
Given that Java 8 is EOL it might make sense to drop it from testing

Reply:
it's not really, each vendor can choose how long they will support it for, oracle has chosen to stop doing updates for commercial use although personal and developer updates are still there for free,

adoptopenjdk is saying 2026 for their distribution: https://adoptopenjdk.net/support.html

When Jenkins drops java 8 as a requirement (hopefully sometime next year) then it would make sense to do this but not yet

https://groups.google.com/g/jenkinsci-dev/c/nEw4TLDcZLw/m/x5KJ0WVFAAAJ

Reply:
Makes sense, thanks for the reference